install_config/router/f5_router.adoc

```diff
@@ -133,7 +134,8 @@ BIG-IP®* host:
 address.
 
 |`--external-host-username`
-|Specifies the *F5 BIG-IP®* user name (typically *admin*).
+|Specifies the *F5 BIG-IP®* user name (typically *admin*). 
+The *F5 BIG-IQ* user account must have access to the Advanced Shell (Bash) on the F5 BIG-IP system.
```

Comment:
Seems  here ```The *F5 BIG-IQ* user account``` has not be updated to `BIG-IP`. I checked the F5 doc and found that BIG-IQ is Centralized Management system and can discover, manage and monitor remote BIG-IP devices, but it is not needed for OpenShift and F5 integration, so I think better to use `BIP-IP` here avoiding confusion.

Reply:
@lihongan Thank you for clarifying. I don't think I saw that I was using the wrong term.